admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java

```diff
@@ -114,7 +116,7 @@ public ApplicantDetailsDto getApplicantDetails(String rid) throws Exception {
                     LinkedHashMap<String, String> jsonObject = utility.getJSONValue(mapperIdentity, valueObj);
                     String value = jsonObject.get(VALUE);
                     applicantDataMap.put(value,identityObj.get(value).toString());
-                } else if (valueObj.equalsIgnoreCase(ApplicantPhoto)) {
+                } else if (valueObj!=null && valueObj.equalsIgnoreCase(ApplicantPhoto)) {
```

Comment:
we follow to use full caps for constants. can we follow the same here?

Reply:
updated the logic as suggested